### ISSUE TITLE
Email PDF isn't sent Magento 2.2.4

### ISSUE THREAD

**Issue by Calebvw** [EaDesgin/magento2-pdf-generator2#48]
The module is not compatible with magento 2.2.4.

**Comment by rishabhchd19** [comment]
Yes. Same problem here. When selected "Email Copy of Invoice" it sent a copy to Admin Email but not a Customer Email. When disabled this extension, Invoice Email sent to both Customer and Admin but without an attachment ofcourse . Debug & System log not showing anything wrong. Probably the extension not working on 2.2.4

**Comment by rishabhchd19** [comment]
Any solution for this???

**Comment by aweb3r** [comment]
Also with Magento 2.1.4

**Comment by nobodyMO** [comment]
I had the same problem after installing the fix from https://github.com/sashas777/bug-from-email to solve the email from bug in magento 2.2.4. 
Both modules overwrite the classes \Magento\Framework\Mail\Template\TransportBuilder and \Magento\Sales\Model\Order\Email\SenderBuilder.
It seems that the sending of the PDF doesn't work anymore if another module changes those classes as well. 
Perhaps this will help to identify the reason.

**Comment by Calebvw** [comment]
Does anyone know an extension that does work?
